mapper/src/main/java/org/jboss/pnc/mapper/api/BuildConfigurationMapper.java

```diff
@@ -87,6 +88,7 @@ default BuildConfiguration toIDEntity(BuildConfigurationRef dtoEntity) {
     @Mapping(target = "parameters", source = "genericParameters")
     @Mapping(target = "creationUser", qualifiedBy = Reference.class)
     @Mapping(target = "modificationUser", source = "lastModificationUser", qualifiedBy = Reference.class)
+    @Mapping(target = "brewPullActive", source = "brewPullActive")
```

Comment:
this shouldn't be required if they have the same name

Reply:
OK